tests/cluster_tests/test_pbspro.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
```

Comment:
Good point, I added some comments to it.  It is meant to be a more generic test than test_qsubs.sh, which includes INL specific things like module load raven-devel-gcc. I tested test_pbspro.sh and it failed by default, but worked once I created $HOME/.raven/environments/raven_libs_profile and $HOME/.raven/default_runinfo.xml